app/src/cobrand/res/values/themes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <style name="AppBaseTheme.Splash">
+    <item name="android:windowBackground">@drawable/splash_screen</item>
+  </style>
+</resources>
```

Comment:
Do we actually need this file? I thought that having just a themes.xml in main would be enough, and then splash_screen drawable would differ between flavours.

Reply:
SVG's doesn't seem to be supported as a bitmap source... I was getting errors because of it, hence why I used the pngs.

I think you are right about this cobrand file, I think that was some experiementation I was doing with different drawables and then I forgot to merge this file back into main, will test it now.